packages/ts/react-crud/test/autocrud.spec.tsx

```diff
@@ -108,11 +108,11 @@ describe('@vaadin/hilla-react-crud', () => {
 
     it('can hide new button', async () => {
       const { rerender } = render(<AutoCrud service={personService()} model={PersonModel} />);
-      await waitFor(() => expect(screen.queryByText('+ New')).to.exist);
+      await waitFor(() => expect(screen.queryByText('+ New')).to.have.style('visibility', 'visible'));
```

Comment:
I'd suggest using `chai-dom` here (may require `use(chaiDom)` directive):
```suggestion
      await waitFor(() => expect(screen.queryByText('+ New')).to.be.displayed;
```

Reply:
If using `visibility: hidden`, `to.not.be.displayed` fails

Reply:
Yes, for `visibility: xxx` you need `to.be.visible` instead